docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -47,19 +47,19 @@ The AWS JDBC Driver also has a parameter, [`wrapperLoggerLevel`](#aws-advanced-j
+| `password`                        | `String`  | No       | Database password.                                                                                                                                                                                                                                                                            | `null`        |
+| `wrapperDialect`                  | `String`  | No       | Please see [this page on database dialects](/docs/using-the-jdbc-driver/DatabaseDialects.md), and whether you should include it.                                                                                                                                                              | `null`        |
+| `wrapperLogUnclosedConnections`   | `Boolean` | No       | Allows the AWS JDBC Driver to capture a stacktrace for each connection that is opened. If the `finalize()` method is reached without the connection being closed, the stacktrace is printed to the log. This helps developers to detect and correct the source of potential connection leaks. | `false`       |
+| `loginTimeout`                    | `Integer` | No       | Login timeout in milliseconds.                                                                                                                                                                                                                                                                | `null`        |
+| `connectTimeout`                  | `Integer` | No       | Socket connect timeout in milliseconds.                                                                                                                                                                                                                                                       | `null`        |
+| `socketTimeout`                   | `Integer` | No       | Socket timeout in milliseconds.                                                                                                                                                                                                                                                               | `null`        |
+| `tcpKeepAlive`                    | `Boolean` | No       | Enable or disable TCP keep-alive probe.                                                                                                                                                                                                                                                       | `false`       |
+| `wrapperTargetDriverAutoRegister` | `Boolean` | No       | Allows the AWS JDBC Driver to register a target driver based on `wrapperTargetDriverDialect` configuration parameter or, if it's missed, on a connection url protocol.                                                                                                                        | `true`        |
```

Comment:
i don't think we should use or add the `wrapper` prefix to any property

`targetDriverAutoRegister` or even just `autoRegister`